src/Components/Book.css

```diff
@@ -0,0 +1,178 @@
+.title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: "RobotoSlab", sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+}
+
+.card-container {
+  width: 100%;
+  height: 10.625rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 0 1.688rem 5%;
+  border-radius: 4px;
+  border: solid 2px #e8e8e8;
+  background-color: #fff;
+}
+
+.card-container .card {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: "RobotoSlab", sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.btn-card {
+  width: 4.563rem;
+  height: 1.188rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-family: "RobotoSlab", sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  background: none;
+  border: none;
+}
+
+.card-progress {
+  display: flex;
+}
```

Comment:
- [ ] Kindly add the `align-items: center` to center this progress-card vertically. check the difference here :point_down: 

Your card-progress | The template card progress
![image](https://user-images.githubusercontent.com/65068771/155845076-24bf880e-6448-4265-8cab-9a90493ce07e.png)  ![image](https://user-images.githubusercontent.com/65068771/155845103-1bdedeb1-9135-4d5c-9018-93cc60c6f276.png)

- [ ] Use also the corresponding color for the "Completed" text still in the card-progress section.

- [ ] You may also add the border at the right of the card-progress as it is the template.

Reply:
Done